src/webapi/node_list.rs

```diff
@@ -30,6 +30,16 @@ impl NodeList {
         js!( return @{self}.length; ).try_into().unwrap()
     }
 
+    /// Returns a node from a NodeList by index.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/item)
+    // https://dom.spec.whatwg.org/#dom-nodelist-item
```

Comment:
This should link to the IDL: `https://dom.spec.whatwg.org/#ref-for-dom-nodelist-item`